sanitizers/src/main/java/com/code_intelligence/jazzer/sanitizers/RegexInjection.kt

```diff
@@ -49,7 +49,7 @@ object RegexInjection {
     fun compileWithFlagsHook(method: MethodHandle, alwaysNull: Any?, args: Array<Any?>, hookId: Int): Any? {
```

Comment:
Link in `CANON_EQ_ALMOST_EXPLOIT` docs to this method is broken.

Reply:
Fixed.